scripts/requirements.txt

```diff
@@ -0,0 +1,5 @@
+requests==2.19.1
+numpy==1.15.2
```

Comment:
This package and some below are copied from `requirements.txt` in `examples/grpc_client`. I my opinion you should consider nested `requirements.txt` file. This will allow to not maintain two files in case of changes.